test/XmlValidationTest.php

```diff
@@ -32,7 +32,8 @@
 namespace OpusTest\Import;
 
 use DirectoryIterator;
-use Opus\Import\XmlValidation;
+use Opus\Import\Xml\MetadataImportInvalidXmlException;
+use Opus\Import\Xml\XmlValidator;
```

Comment:
Die Test-Klasse sollte in ihrem Namen und ihrer Position, die getestete Klasse spiegeln.